tfrog-motordriver/main.c

```diff
@@ -603,7 +658,11 @@ int main()
   r_rs485buf_pos = 0;
   printf("RS485 init\n\r");
   AT91C_BASE_PMC->PMC_PCER = 1 << AT91C_ID_US0;
-  USART_Configure(AT91C_BASE_US0, AT91C_US_USMODE_RS485 | AT91C_US_CHRL_8_BITS, 3000000, BOARD_MCK);
+
+  if (HEARTBEAT)
+    USART_Configure(AT91C_BASE_US0, AT91C_US_USMODE_RS485 | AT91C_US_CHRL_8_BITS, 115200, BOARD_MCK);
```

Comment:
On 115200bps, RS485 daisy chain mode is almost useless due to insufficient data rate.
Might be better to be changed by EEPROM data.